news/7968.bugfix

```diff
@@ -0,0 +1 @@
+Use custom ``call_subprocess`` for vcs commands to avoid merging stdout and stderr,  and introduce a new ``SubprocessError`` exception
```

Comment:
This isn't very relevant to end users, who don't care about pip's internals. Can we rephrase this to be better suited for the Changelog?

Reply:
How about something like `The VCS commands run by pip as subprocesses don't merge stdout and stderr anymore, improving the output parsing by subsequent commands`

Reply:
Have updated the news entry according to my above suggestion.